api/client.go

```diff
@@ -333,6 +333,17 @@ func (c Client) DeleteProvider(ctx context.Context, id uid.ID) error {
 	return delete(ctx, c, fmt.Sprintf("/api/providers/%s", id), Query{})
 }
 
+func (c Client) ListCredentialRequests(ctx context.Context, req ListCredentialRequest) (*ListCredentialRequestResponse, error) {
+	return get[ListCredentialRequestResponse](ctx, c, "/api/credentials", Query{
+		"destination":     []string{req.Destination},
```

Comment:
Looks like we will want to re-name the existing Credentials struct that stores password hashes for users